mod-vendors/schemas/vendor.json

```diff
@@ -167,7 +167,167 @@
     "edi": {
       "description": "The EDI object for this vendor",
       "type": "object",
-      "$ref": "edi.json"
+      "properties": {
+        "vendor_edi_code": {
+          "description": "The vendor code for this EDI",
+          "type": "string"
+        },
+        "vendor_edi_type": {
+          "description": "The vendor type for this EDI",
+          "type": "string"
+        },
+        "lib_edi_code": {
+          "description": "The library code for this EDI",
+          "type": "string"
+        },
+        "lib_edi_type": {
+          "description": "The library type for this EDI",
+          "type": "string"
+        },
+        "prorate_tax": {
+          "description": "The setting to prorate tax for this EDI",
+          "type": "boolean"
+        },
+        "prorate_fees": {
+          "description": "The setting to prorate fees for this EDI",
+          "type": "boolean"
+        },
+        "edi_naming_convention": {
+          "description": "The naming convention for this EDI",
+          "type": "string"
+        },
+        "send_acct_num": {
+          "description": "The setting to send the account number for this EDI",
+          "type": "boolean"
+        },
+        "support_order": {
+          "description": "The setting to support order for this EDI",
+          "type": "boolean"
+        },
+        "support_invoice": {
+          "description": "The setting to support invoice for this EDI",
+          "type": "boolean"
+        },
+        "notes": {
+          "description": "The notes for this EDI",
+          "type": "string"
+        },
+        "edi_ftp": {
+          "description": "The FTP object for this EDI",
+          "type": "object",
+          "properties": {
+            "ftp_format": {
+              "description": "The FTP format for this EDI",
+              "type": "string"
+            },
+            "server_address": {
+              "description": "The server address for this EDI",
+              "type": ["string", "null"],
+              "pattern": "^([Hh][Tt][Tt][Pp]|[Ff][Tt][Pp])([Ss])?://.+$"
+            },
+            "username": {
+              "description": "The login username for this EDI",
+              "type": "string"
+            },
+            "password": {
+              "description": "The login password for this EDI",
+              "type": "string"
+            },
+            "ftp_mode": {
+              "description": "The FTP mode for this EDI",
+              "type": "string"
+            },
+            "ftp_conn_mode": {
+              "description": "The FTP connection mode for this EDI",
+              "type": "string"
```

Comment:
according to https://docs.google.com/spreadsheets/d/1F88PoSnEKD7oFnza1ZuOgzAgKnK95XOyTQzxyww3tZA/edit?ts=5c1bd54f#gid=0 this can/should be made into an enumeration Active/Passive

Reply:
changed to enum